### DISCUSSION TITLE
Activate Esim R Link 1

### DISCUSSION THREAD

**Discussion by sv3pjl** [cedricp/ddt4all#953] [Q&A]
Hello, i have a Captur I, first registered in Netherlands. It's a 2016 model so all the online services are not supported anymore.
After a contact with official Renault they said that i cannot have online services anymore. I know that r link 1 have an esim chip and through ddt4all i can configure some provider parameters. Orange provide some packets for esim but only for openRlink devices. Does anyone managed to activate the esim with a mobile operator someway?

**Comment by AndyIreland** [comment]
What does the E stand for in esim? - Enhanced? External? 

anyway what I would be doing is checking all the parameters are enabled in the DCM / TCU ECU and the radio settings and the mobile provider settings (on my rlink1 on my renault Zoe EV the MNC/MCC provider is set as 208/02 (orange France) even though Zoe was first registered in the UK

**Reply by sv3pjl** [reply]
well, it stand for embedded SIM card, not a physical SIM card. There is not any slot to change the sim card. It's a chip. In connectivity information shows it is disconnected. I checked all the parameters in ECU.

**Reply by AndyIreland** [reply]
ah yes thats right , embedded . 

if you want me to have a look over the internet (sometimes another person having a look might find something you didnt) at your ECU Settings just let me know 

You will need:

a Bluetooth or USB elm327 (not wifi) adaptor,
a windows 7,8,10,11 laptop
Internet connectivety on your laptop in way of signal from home router of mobile hotspot,
a specially adapted cable to read the hi CAN for the  r-link settings,

Sometimes (in case of Zoe) sometimes people have got back connectivety by removing fuse for TCU and waiting for a few seconds and putting back in. - some times it gets it re-connected.
some times people have got their rlink connected by getting in touch with rlink UK technical support, and sometimes this is the info they may give you to try: 

a) On the R-Link's Main Menu, access Services
b) Tap on Settings
c) Select Data Sharing Settings
d) You will have four options: No / Help / Yes / Language
e) The first time, select No, and accept all subsequent prompts
f) Go back to the same screen as above and this time select Yes
g) Accept any prompts that might come up
h) Completely switch off the device, leave the vehicle and wait for a few
minutes. Afterwards, switch it back on.
i) Go to Menu > System > Status and Information > Network Status and see
if it now shows Connected.

it works for some people, it might be worth you giving a go - i know you have already been in touch with them and they said you cannot have online services any more, but maybe they are referring to the tom-tom live service or map update services you are not entitled to now , so its maybe worth a try to get the full bars with 'connected' underneath trying these tips?

Also, on your R-Link1 what operating software are you running are you up to date for your rlink? - you want to be showing a figure like 11.346xxx or 11.347xxx on screen.

**Reply by sv3pjl** [reply]
thanks Andy, i checked all, software is up to date. I will try all you said again but i'm sure i have done all your suggestions in the past. Last info i post the ddt4all  tcu info screen
![1701710205815](https://github.com/cedricp/ddt4all/assets/57438190/619bd4af-e060-44b5-a54a-34bac7fd2e93)
![1701710205838](https://github.com/cedricp/ddt4all/assets/57438190/54c74226-249f-49ed-89c6-ed9c7f714b95)

**Comment by AndyIreland** [comment]
good sign that you have the signal bars lit - Vodafone GR , thats Greece is it?
have you checked these in the MFD settings and see what they are filled out with?
![image](https://github.com/cedricp/ddt4all/assets/49172217/500afc4a-5a1b-4bf5-8a3a-186987ee52ad)

**Reply by sv3pjl** [reply]
yes it Greece Vodafone

**Comment by AndyIreland** [comment]
and these all filled in ok on the TCU ?
![image](https://github.com/cedricp/ddt4all/assets/49172217/b66e57d4-0ef8-449c-a771-6c490d3e8d0b)

**Reply by sv3pjl** [reply]
all parameters all filled, i think that the operator don't register my esim due to end of 3 years connectivity. That's why i'm trying to find another solution.
![1701715204961](https://github.com/cedricp/ddt4all/assets/57438190/a0eaa81d-ee97-4fcd-a9c7-c1925be33d06)
![1701715271646](https://github.com/cedricp/ddt4all/assets/57438190/1905b6dd-beab-4460-8af5-42dccd8ed2e6)

**Reply by matzel82** [reply]
The problem is, that the SIM is disabled at the Renault server. The service have to be payed!!!

**Reply by matzel82** [reply]
Your sim have to register to Renault to get access. It's known active service. I know this really good because I'm work at Renault and I'm Renault Multimedia Expert.

**Reply by sv3pjl** [reply]
i have contacted Renault and they say that this service is not available anymore for my vehicle.

**Reply by AndyIreland** [reply]
The problem is, that the SIM is disabled at the Renault server. The service have to be payed!!!

So in the case of my Renault Zoe EV I have a current subscription of Connected Services Z.E. subscription - so in the case of the Captur 2016 what is the equivalent subscription?

**Reply by sv3pjl** [reply]
The renault does not give me the option of subscription. That's the problem, if i had this option i would have paid for it.

**Reply by AndyIreland** [reply]
In Greece is it in you now (or is it netherlands??) 
If in Greece have you gone on this site  to have a look and see if can be purchased from here?
https://gr.rlinkstore.com/registration/my-profile

**Reply by matzel82** [reply]
> i have contacted Renault and they say that this service is not available anymore for my vehicle. 

It's a long time inactive.... that's it...

**Reply by sv3pjl** [reply]
so what do you suggest? is there any way to enable it?

**Reply by matzel82** [reply]
Sorry....the time is over...

**Comment by AndyIreland** [comment]
thats a bit strange all the digits in your APN & Password etc on your tcu settings: 
![image](https://github.com/cedricp/ddt4all/assets/49172217/94a46c4f-c859-4f98-ab09-e2b019cbb904)



I reckon it should have settings like this instead
![image](https://github.com/cedricp/ddt4all/assets/49172217/135efbe6-6708-43aa-b650-c44687fc5105)

With nocatcp2.fr.fg or nocatcp.fr.fg 
and RCS1978 as the password 
(which is settings on my Zoe TCU)

and tcugateway2 renault.com/evsenderservice/tcuservice or tcugateway renault.com/evsenderservice/tcuservice (on my Zoe TCU)

**Comment by AndyIreland** [comment]
> Hello, i have a Captur I, first registered in Netherlands. It's a 2016 model so all the online services are not supported anymore. After a contact with official Renault they said that i cannot have online services anymore. I know that r link 1 have an esim chip and through ddt4all i can configure some provider parameters. Orange provide some packets for esim but only for openRlink devices. Does anyone managed to activate the esim with a mobile operator someway?

Just by matter of fact @sv3pjl  what connected service do you want? - is it the TomTom Live services you want to use? 
I know on my EV with r-link1 if I want connected services I have to have a EV connect subscription which is about 25eur per year I think , and then TomTom live subscription for around 56eur per year I think it it is (my TomTom run out and I didnt bother to renew it)

**Reply by sv3pjl** [reply]
If i had online access then i would try to build an application for remote manage playing with the gateway, similar to connected services. I don't need tomtom live subscription.The idea is to build a private server but that need online access.

**Reply by AndyIreland** [reply]
Do you fancy giving me your VIN number of your car (even private message it) so I can check on my rlink store account and see if there is a subscription listed for it/yours? - see I would have to add it as another car in 'my garage' section. 
I dont really know how you could add your own server, think it only works with that tcugateway server only from renault. Most probably a special SIM (or chip) only programmed to work with renault servers)

**Reply by sv3pjl** [reply]
thanks andy , i have already checked that. I can add my vehicle but it Is says that your current vehicle does not support one of the service packages Ζ.E. online or easy connect.

**Reply by AndyIreland** [reply]
Thats OK , was just thinking sometimes different subscription packages can be available on different rlink store sites for different countries. I remember once, more subscriptions/apps were on the UK/GB rlink store website that on the irish rlink store website, so I was just curious if maybe the connectivety service subscription showed up for my rlink store that wasnt showing up when you put your VIN for your rlink regional store .
It was just a thought

My idea was that if it did show up as a subscription for me I could subscribe my end then (because it seems to be tied to SIM/TCU) it might enable it for you  (I dunno by sending a signal to tour car/whatever way it does it)

**Reply by sv3pjl** [reply]
i've already  tested with different countries... no luck

**Reply by AndyIreland** [reply]
oh well thats good, at least you tried

**Comment by AndyIreland** [comment]
Your rlink tech support might not be as friendly as our one in the UK but I remember one time (I bought my 2013 Zoe EV in 2020) and i remember this one time I went on the UK rlink store website and i could not see the ze services listed on the site. I emailed the UK rlink email helpline and they said something about something to do with Brexit or something and it was not available to buy through the UK rlink site .... but "as a gesture of goodwill I will apply a years subscription free of charge to you and check back in a few hours time and it should connect" (so I could do things like remote charging and precon heating through the app which sent a signal to the onboard rlink) 
anyway 2 hours later it kept to his word and I looked and there were all fully lit signal bars and the word 'connected' underneath whereas before there was the word 'disconnected' !

**Comment by AndyIreland** [comment]
is there anything helpful to you in this thread @sv3pjl  , a lot of it went over my head but if you are interested hacking into your TCU and servers and that you might understand it all and how it works? - its for a Zoe EV but it should be same TCU and r-link1 as in your captur
https://www.speakev.com/threads/telematics-hacking-is-it-possible.131872/

**Reply by sv3pjl** [reply]
unfortunately no luck, the above thread is what i have in mind but i have no luck with connectivity

**Comment by AndyIreland** [comment]
seems crazy that you cannot renew your connectivity features - your even willing to pay for it! - I wonder what the real reason is, I know the subscription runs out , but there should be a way of renewing it. 
Heres what shows up available for my ZOE EV on my rlink store site: 
![image](https://github.com/cedricp/ddt4all/assets/49172217/7033d6d8-f22e-413e-a2c9-ff62e9b4412d)


Have you looked at the  r-link store app screen on the r-link1 screen itself to see if connectivity subscription shows up?

**Reply by sv3pjl** [reply]
here is what shows mine from the UK store 
![Χωρίς τίτλο](https://github.com/cedricp/ddt4all/assets/57438190/2e2c4b9f-7ea6-4b81-ad90-e73ef18fb5cc)

Of course the same is from gr store and nl store.

**Reply by AndyIreland** [reply]
yeah i see - looking on well the kadjar forum anyway, you get that message if the car has not been subscribed to any subscriptions in 3 years and looks like renault disable the sim . 
presuming the ext SIM is not a replaceable one , that would mean I guess changing the whole TCU unit with one that has not had its sim disabled

**Reply by sv3pjl** [reply]
well this could be a solution but i am not willing to do that...

**Reply by AndyIreland** [reply]
no I don't blame you - cost a hell of a lot.

Did this ever go to 'registered' or did it just stay on 'searching'?
![image](https://github.com/cedricp/ddt4all/assets/49172217/bc566cd6-a630-48c8-ba97-b7574d47d85c)
 

If its true and the SIM (EXT SIM) has indeed been disabled by renault I wouldnt have thought it would even be at the searching stage and full bars on the screen

**Comment by AndyIreland** [comment]
@sv3pjl  - do you see here on this post on jan6 2022 and they said that their sub run out in 2017 and they got in touch with support and it was reactivated in 72hours!
![image](https://github.com/cedricp/ddt4all/assets/49172217/eda849b3-393d-443f-a73b-c388157c3b87)

https://www.speakev.com/threads/r-link-connection-status-and-live-services.165392/

**Comment by andriy-nakonechnyy** [comment]
Hello. I have the same issue as described by @sv3pjl . My car is espace 2016. On previous car (grand scenic 2012) with TomTom live I replaced gsm module with sim card slot and everything works fine now.
If you have fixed this trouble on R-link please write

**Reply by BrianSPC** [reply]
Is there a sim card inside the TomTom om tage Scenic 3 ?   And did you just switch it with a normal data sim card ?

**Reply by andriy-nakonechnyy** [reply]
There is a small gsm chip on the board, you need remove it and solder sim card holder for Samsung P1000 GalaxyTab/P3100/P3110/P6200/P7100/N8000. After you can use sim card and buy subscribe for live services. This works perfect for my Renault Grand Scenic 3.
This board is located on the left near the central display
![FiAAAgAtn-A-1920](https://github.com/cedricp/ddt4all/assets/44396853/c3ed8f24-952b-4c4e-9981-5bce1aad45cb)
![8YAAAgAtn-A-1920](https://github.com/cedricp/ddt4all/assets/44396853/b3c86718-e8a4-484c-ad23-7583ca2e63dd)

**Reply by BrianSPC** [reply]
Thanks for the info. Is it behind the TFT car display or TomTom display ?

and is it easy to get the display out of the cockpit?

**Reply by BrianSPC** [reply]
I have ordered a SIM card holder 👍 and then I can just put in a normal data SIM card ?

**Reply by andriy-nakonechnyy** [reply]
It is behind the TomTom display.[You can use this manual](https://www.youtube.com/watch?v=opPcmBtrYXg)

**Reply by andriy-nakonechnyy** [reply]
Yes, if you put there sim card holder you can use normal sim card

**Reply by BrianSPC** [reply]
I have now done the install, and it have found my normal sim card and have full bars, but cant Connect to live services,  what should I do ? 
![Uploading IMG_0985.jpeg…]()

**Reply by AndyIreland** [reply]
your image did not upload. 
But in any case first of all make sure you are currently paid up/subscribed to TomTom live - if you are you may need to activate TomTom live in the services of your TCU:
![image](https://github.com/cedricp/ddt4all/assets/49172217/9a3c4984-7154-46aa-ac17-3a203aba3dd8)

**Reply by BrianSPC** [reply]
What software are you using for that ?      I have RenoLink can I use that ?

**Reply by AndyIreland** [reply]
I wouldn't know sorry, i have never used RenoLink - this is a DDT4ALL issue forum you posted on so I had presumed you were using DDT4ALL for your programming the ECU's

**Reply by BrianSPC** [reply]
Hi, I have now DDT4ALL working, but I cant find the TCU,   What ECU/Car should i chose ?   I have a Grand Scenic 3 from 2012,   it is not on the list.....            so is it Ranault X96 or what should I do ?

**Reply by AndyIreland** [reply]
you might have to select it manually - try selecting one of these , one of them might do. 
If you read and you have no boxes with 'NO DATA' in them you might be OK , but be careful when selecting ECU's blindly manually , do and change parameters at your own risk
![image](https://github.com/user-attachments/assets/0d7886f5-9827-4d8b-8085-520fe3c58209)

**Comment by akmadov** [comment]
@AndyIreland Sir, would you be willing, when and if you have the time, to send me a few screenshots of your 2013 Zoe TCU configurations in DDT4ALL. The problem I am facing is that the data is not updated in the MyRenault app. First I had the data hang on the last entry from March 1st 2024; afterwards, at some point in time, it was updated on May 21st and has remained so since. On the Rlink I can see status connected and it shows IMEI but no SIM ID (I do not remember it showing SIM ID at any point in time). In DDT4ALL I was able to see that the TCU was in roaming mode (registered to communication network field) and it shows it is connected to a network provider in my country (originally the car was sold in Switzerland but I bought it in Bulgaria where it resides currently). I am trying to figure out whether I will need to buy a second hand TCU and try and replace it or it is some sort of issue with the TCU configuration, possibly resulting from some changes on the network provider's side.
In any case, I would be very grateful for any information you are willing to give me! I am using the TCU_GEN2_V3.2 ECU in DDT4ALL to view said data.

**Reply by AndyIreland** [reply]
hello there - exactly the same on my rlink1 in my renault Zoe - it has never ever shown SIM ID on the screen and has always been blank. 
![image](https://github.com/cedricp/ddt4all/assets/49172217/d1591e46-2843-4261-8821-e52391a38218)

But yes the same as yours IMEI shows up and as long as I have a subscription and if MCC/MNC is filled out and signal bars are lit and 'Connected' is underneath and if the Renault Servers are working properly (which a lot of time they dont appear to be) then it works between car and my renault app. 

So really question I have to ask you:
Have you got valid subscription or has it expired/run out?
Have you got full lit signal bars?
Have you got word 'CONNECTED'  underneath signal bars?

**Reply by akmadov** [reply]
My subscription is valid for at least 1.5 more years. The signal bars are either 3/4 or full most of the times and the status is connected.
Before March 1st, as noted, I haven't had reliability issues with the service and all of a sudden this started happening. Unfortunately Renault dealers in my country tell me that this service is not officially offerered for 2013 Zoes so there is nothing they could do.
I am wondering if the missing SIM ID is the result of the "roaming mode", as from what I understand this is your case as well.
Regardless, I am wondering if I can get a second hand TCU and try to temporarily install in somehow using the "white" connector behing the Rlink display so that I can check its operations before ripping out the dashboard to replace it entirely. Also, I am wondering if and what configuration will the new TCU need (maybe apart from writing the VIN of my car to the new TCU). Thank you for your response!

**Comment by Adamyno** [comment]
I have a similar problem. There has been no communication with the servers since March 1, 2024. I have a valid subscription. I added the vehicle again in the MyRenault app, now the status on the screen is: connected. The r-link store works in the car, it loads both my current and expired subscriptions, you can see how it loads the icons one after the other, so there is real-time communication, but the vehicle status is not updated within the app.

Someone said that the service will be discontinued and the TCU software needs to be updated, which Renault supposedly does for free. I am in Hungary, I will ask the brand services, but to be honest, I don't believe it anymore. They don't give anything for free :)

**Comment by AndyIreland** [comment]
the mobile providers are gradually turning off 2G/2.5G (EDGE) and 3G in some countries now to give more bandwidth to 4G and 5G - that will be challenging for people with the older Zoe's such as 22kw & ZE40 which I believe use 2.5G EDGE TCU for connectivity.

**Reply by Skolekridt** [reply]
Hi Andy.
Yes, here in Denmark it has been off for almost 1 year. And just bought a Renault Zoe from 2016, where it will never be able to connect again. :( And just discovered that staggered charging (Calendar) is part of their subscription. And when you can't pay the subscription and be online. Then you can't use the calendar function to set up charging when it's cheapest.
You or others haven't found a way to activate that function through DDT4ALL? I hope so, because I really want to use that function. Thanks in advance for your help.

Sorry, I know this is an old thread. But I hope this is okay. :)

**Reply by AndyIreland** [reply]
> Hi Andy. Yes, here in Denmark it has been off for almost 1 year. And just bought a Renault Zoe from 2016, where it will never be able to connect again. :( And just discovered that staggered charging (Calendar) is part of their subscription. And when you can't pay the subscription and be online. Then you can't use the calendar function to set up charging when it's cheapest. You or others haven't found a way to activate that function through DDT4ALL? I hope so, because I really want to use that function. Thanks in advance for your help.
> 
> Sorry, I know this is an old thread. But I hope this is okay. :)

Hi there.
the only thing I can think of is to go into the TCU settings of your zoe in DDT4ALL and make sure charge schedule is activated and see what happens then, see if it comes to life after that. but it might rely on the connection to renault servers and a connection and subscription. if it does there is not anything more you can do. 
Good luck.

<img width="1345" height="649" alt="image" src="https://github.com/user-attachments/assets/7d021459-2af0-49f1-b7c9-233add94e17e" />
